output.go

```diff
@@ -76,12 +76,12 @@ func NewOutput(tty io.Writer, opts ...OutputOption) *Output {
 	for _, opt := range opts {
 		opt(o)
 	}
-	if o.Profile < 0 {
-		o.Profile = o.EnvColorProfile()
-	}
 	if o.tty == nil {
```

Comment:
This needs to be moved above the `opts` range loop. Actually. This is where we could end up querying the current color values already (`WithColorCache`).